site/docs/flink.md

```diff
@@ -78,12 +81,25 @@ as the following:
 # HADOOP_HOME is your hadoop root directory after unpack the binary package.
 export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 
+# download Iceberg dependency
+ICEBERG_VERSION=0.11.1
+MAVEN_URL=https://repo1.maven.org/maven2
+ICEBERG_MAVEN_URL=$MAVEN_URL/org/apache/iceberg
+ICEBERG_PACKAGE=iceberg-flink-runtime
+wget $ICEBERG_MAVEN_URL/$ICEBERG_PACKAGE/$ICEBERG_VERSION/$ICEBERG_PACKAGE-$ICEBERG_VERSION.jar
+
 # wget the flink-sql-connector-hive-2.3.6_2.11-1.11.0.jar from the above bundled jar URL firstly.
```

Comment:
Nit: Can we not use the hard coded file names in the jar if we're making all of that into arguments?

Reply:
done